examples/package.json

```diff
@@ -61,6 +61,7 @@
         "lodash": "^4.17.21",
         "react": "^18.3.0",
         "react-dom": "^18.3.0",
+        "react-router": "^7.0.2",
```

Comment:
are a handful of static, independant pages worth pulling in this dependency?

Reply:
I suppose it does let us delete the kinda gross repetetative {demo-name.html} files....

Reply:
lets keep it, seems fine